src/nupic/algorithms/KNNClassifier.py

```diff
@@ -201,6 +207,7 @@ class labels
     self.replaceDuplicates = replaceDuplicates
     self.cellsPerCol = cellsPerCol
     self.maxStoredPatterns = maxStoredPatterns
+    self.minSparsity = minSparsity
```

Comment:
Thanks.  That, or fix it in Imbu, which I think would involve retraining and serializing models.